src/components/Button/index.jsx

```diff
@@ -1,13 +1,26 @@
+import GoogleButtonIcon from "../GoogleButtonIcon/GoogleButtonIcon";
 import "./styles.scss";
 import PropTypes from "prop-types";
 
 const Button = (props) => {
-  return <button type={props.type} className='button'>{props.text}</button>;
+  return (
+    <button type={props.type} className={`button ${props.styleGoogle}`}>
+      {props.styleGoogle ? (
+        <span className="button__img">
+          <GoogleButtonIcon />
```

Comment:
Pass icon as a prop instead of hardcoding `GoogleButtonIcon`. Allows reuse with different icons, .e.g. Google, Facebook, Apple, etc.